etcdserver/etcdserverpb/raft_internal_stringer_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcdserverpb
```

Comment:
nit: package etcdserverpb_test

as it seems you depend only on public API of that module (etcdserverpb).

https://tip.golang.org/cmd/go/#hdr-Test_packages

Reply:
@ptabor Thank you for the review. I've updated the code as you've suggested.